docs/src/main/tut/typeclasses/lawtesting.md

```diff
@@ -0,0 +1,146 @@
+# Law testing
+
+[Laws](https://typelevel.org/cats/typeclasses.html#laws) are an important part of cats.
+They help us enforce constraints that we usually assume to be true and give us guarantees that our code does what we think it does.
+Manually testing each data type to adhere to each of its type class instances can be quite cumbersome.
```

Comment:
hmm, I think the wording here could use some polish.  e.g. it's not testing data types, it's testing instances.  I'd say for the sake of conciseness and time, how about we just say 
```
[Laws](https://typelevel.org/cats/typeclasses.html#laws) are an important part of cats.
Cats uses catalysts and dicipline to help test instances with laws. 
```
then introduce cats-testkit